playbooks/collect_activity/stats.py

```diff
@@ -1,4 +1,4 @@
-#!/edx/bin/python.edxapp
+#!/edx/app/edxapp/venvs/edxapp/bin/python
```

Comment:
@0x29a, why is this change needed? Doesn't `/edx/bin/python.edxapp` use the same virtualenv environment?

Reply:
@lgp171188, AFAIK `python.edxapp` [doesn't exist](https://discuss.openedx.org/t/koa-installation-attempt-missing-python-edxapp/3872/2?u=0x29a) in Koa. [Original Shimul's commit](https://github.com/open-craft/opencraft/commit/07b7857dce0974b0af56dcfd01100005d384e92f).